### ISSUE TITLE
Station list not readable in OSX Contract Increase (accessibility) mode

### ISSUE THREAD

**Issue by alexalesi0** [HermesApp/Hermes#291]
i really enjoy OS X with Accessibility mode: Increase Contrast enabled... everything is easier to read but Hermes does not handle the option very well (at least for the station listing list view...).

 see screenshot below 

<img width="793" alt="screenshot 2017-02-23 15 46 31" src="https://cloud.githubusercontent.com/assets/8690050/23278466/aa3550d6-f9df-11e6-963c-6654b95fc5f1.png">

**Comment by alexalesi0** [comment]
oh shoot-this might be fore the weird version of this by "wang wenxing" on App Store. He published what i thought was Hermes called "Music Streaming for Pandora Radio" https://itunes.apple.com/us/app/music-streaming-for-pandora-radio/id1123406334?mt=12

i guess this is different from straight up Hermes. Well, in any case finding his version introduced me to Hermes so it guess its all working out.

**Comment by nriley** [comment]
Yeah, that person is using (and making money from) Hermes’ source code without proper attribution in violation of the license agreement.  I don't test with that mode enabled so if there are any issues with the "real" Hermes please file a ticket.  Thanks!
